.travis.yml

```diff
@@ -27,44 +36,12 @@ matrix:
        apt:
          packages:
            - latexmk
-           - texlive
```

Comment:
I believe `packages:` lists are not additive, so if you want to use `packages:` to install `latexmk`, you need to relist all the other packages here.  Otherwise you can add something like `if [ ! -z "$LATEXMK" ]; then apt-get install latexmk; fi` to the `install:` lines.

Reply:
Thanks, I had just realized that might be the case.